app/scripts/controllers/transactions.js

```diff
@@ -962,17 +962,22 @@ angular.module('openhimConsoleApp')
     //poll for updates for any transactions that are marked as 'Processing'
     //TODO need an endpoint in core to lookup a several transactions by _id at once
     $scope.pollForProcessingUpdates = function() {
-      $scope.transactions.forEach(function(trx) {
-        if (trx.status === 'Processing') {
-          Api.Transactions.get({ transactionId: trx._id, filterRepresentation: 'simple' }, function(result) {
-            $scope.transactions.forEach(function(scopeTrx) {
-              if (scopeTrx._id === result._id) {
-                scopeTrx.status = result.status;
-              }
-            });
-          });
-        }
-      });
+      if ($scope.transactions !== null) {
```

Comment:
Hi @kaweesi nice one on finding the bug.

Instead of wrapping the whole function in an if statement, you could do something like: 
if(!$scope.transactions) { return } at the top of the function.

When testing locally I also found the function $scope.pollForLatest to be writing errors to console when the $scope.transactions was not ready, so can you please add that line to the top of that function too?

Also, double check the indentation.